benchmarks/aes/Dockerfile

```diff
@@ -0,0 +1,92 @@
+# MIT License
+
+# Copyright (c) 2022 EASE lab
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+#---------- PYTHON -----------#
+# First stage (Builder):
+# Install gRPC and all other dependencies
+FROM vhiveease/python-slim:latest as aesPythonBuilder
+WORKDIR /py
+COPY ./benchmarks/aes/python/requirements.txt ./
+RUN pip3 install --user -r requirements.txt
+COPY ./utils/tracing/python/tracing.py ./
+COPY ./benchmarks/aes/python/server.py ./
+COPY ./benchmarks/aes/proto/helloworld_pb2_grpc.py ./
+COPY ./benchmarks/aes/proto/helloworld_pb2.py ./
+
+# Second stage (Runner):
+FROM vhiveease/python-slim:latest as aesPython
+COPY --from=aesPythonBuilder /root/.local /root/.local
+COPY --from=aesPythonBuilder /py /app
+WORKDIR /app
+# ENV PATH=/root/.local/bin:$PATH
+ENTRYPOINT [ "python", "/app/server.py" ]
+
+
+
+
+#---------- GoLang -----------#
+# First stage (Builder):
+FROM vhiveease/golang:1.16-alpine AS aesGoBuilder
```

Comment:
@ustiugov meant you want to make it dependent from vhiveease/golang:latest otherwise it make no sense to have a base image in vhive
However you might need to create a vhiveease/golang-alpine:latest. I dont know exactly if only golang is based on debian or alpine?

Reply:
But instead of using a new alpine image change this dockerfile.
What will not work in debian is the `apk add ...` in line 50,51. Instead you need `apt install`

Reply:
good point. please move to using debian slim in all images. Alpine is more troublesome to work with.

Reply:
Ok forget my last comment the `vhiveease/golang:latest` is an alpine image so it should be fine

Reply:
What I have done right now is pull the golang:1.16-alpine image, tagged it for vhiveease, and used it. Is that fine?

Reply:
no

Reply:
You dont need to anything. Just use `vhiveease/golang:latest` in the docker file should work.